src/utils/coinFormatter.ts

```diff
@@ -52,6 +52,5 @@ export const formatFiat = (amount: BN) => {
 };
 
 export const formatFiatUI = (amount: number) => {
-  const _amount = String(amount).replace(/,/g, '');
```

Comment:
I'm worried that wallet might still be passing a string here even though the type we have defined is number :( 
can we keep this part?